os/drivers/video/video_null_driver.c

```diff
@@ -1481,6 +1600,11 @@ int video_null_initialize(const char *devpath)
 	priv->def_format = &priv->fmt[0];
 	priv->cur_format = &priv->fmt[0];
 	priv->cur_frame = &priv->fmt[0].frames[0];
+	/* Set the default frame interval */
+	priv->cur_fintvl.numerator = 333333;
```

Comment:
Please do not use constant number as possible as you can.

Reply:
Numerator = 333333 and Denominator = 10000000 have no significance in relation to settings for the NULL Driver. These are values used for initialization only for mathematical convenience with fractions, for ease of calculations and to obtain accuracy in the final frame interval calculated.
Similar to Line 1270 `timeperframe.denominator = 10000000;`

Reply:
same opinion as I wrote line 172, null driver should test none-fixed parameter. please provide KConfig or something.

Reply:
Done